test/mock-backend-server/src/main/java/org.wso2am.micro.gw.mockbackend/MockBackEndServer.java

```diff
@@ -32,35 +43,68 @@
     private static final Logger logger = Logger.getLogger(MockBackEndServer.class.getName());
     private HttpServer httpServer;
     private String backEndServerUrl;
-    private static int backEndServerPort;
+    private int backEndServerPort;
     private static boolean retryDone = false;
-
-
+    private boolean secured = false;
+    private boolean mtlsEnabled = false;
 
     public static void main(String[] args) {
```

Comment:
check if an argument could be passed and start tls, mtls backends only for the required test cases.

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/1525/commits/fa0f6674a8e4097018eeb0810dd73ae1a7ef5b90

Reply:
change default backend to tls based server